windows-installer/renew-up.ps1

```diff
@@ -1,12 +1,22 @@
+param(
+    [string]$userLevel = "AllUsers"
+)
+
 function Get-ScriptDirectory { Split-Path $MyInvocation.ScriptName }
 $down = Join-Path (Get-ScriptDirectory) 'renew-down.ps1'
 & $down
 
 $taskName = "Certbot Renew Task"
 
+if ($userLevel -eq "CurrentUser") {
+    $taskUser = $env:UserName
+} else {
+    $taskUser = "SYSTEM"
+}
+
 $action = New-ScheduledTaskAction -Execute 'Powershell.exe' -Argument '-NoProfile -WindowStyle Hidden -Command "certbot renew"'
 $delay = New-TimeSpan -Hours 12
 $triggerAM = New-ScheduledTaskTrigger -Daily -At 12am -RandomDelay $delay
 $triggerPM = New-ScheduledTaskTrigger -Daily -At 12pm -RandomDelay $delay
-$principal = New-ScheduledTaskPrincipal -UserId SYSTEM -LogonType ServiceAccount -RunLevel Highest
+$principal = New-ScheduledTaskPrincipal -UserId $taskUser -LogonType ServiceAccount -RunLevel Highest
```

Comment:
Don't we also need to change thev value for `LogonType` depending on the user?

Reply:
It is better, yes. Done.

Reply:
Note that I keep the logic about CustomUser/AllUsers here, because it is still valid in the context of this script, even if the NSIS installer will never call it using CustomUser currently.

Reply:
But this is now dead code that is never used or tested. I'd prefer we either delete it or continue to let the user continue to choose the installation type rather than leaving the unused code and unnecessary changes to the the NSIS template.

Reply:
Alright. But I put a comment above to remember about adding the logic removed here if we update the installer to allow installation under a specific user.

Reply:
That's great. Thanks for doing that.